shared/elasticsearch-document-adapter/src/contributions/generateLinkedContent.ts

```diff
@@ -45,6 +46,7 @@ export const generateLinkedContent = async (
       title: linkedDocument.title,
     };
   });
-  const linkedContents = await Promise.all(linkedContentPromises);
-  return { linkedContent: linkedContents };
+  return (await Promise.all(linkedContentPromises).then((contents) =>
+    contents.filter((c) => c)
+  )) as ExportFullLinkedContent[];
```

Comment:
```suggestion
  return (await Promise.all(linkedContentPromises).then((contents) =>
    contents.filter((c) => !!c)
  ));
```

Avec cette syntaxe, Typescript détecte que la liste contient des éléments non vide : https://www.benmvp.com/blog/filtering-undefined-elements-from-array-typescript/

Reply:
non j'ai déjà testé j'ai encore l'erreur :(